overlord/configstate/configcore/corecfg.go

```diff
@@ -54,12 +54,7 @@ func coreCfg(tr Conf, key string) (result string, err error) {
 
 // supportedConfigurations will be filled in by the files (like proxy.go)
 // that handle this configuration.
-var supportedConfigurations = map[string]bool{
```

Comment:
IMO having this explicit was nice when grepping around the code.

Reply:
It's either one or the other way, other parts of the code have this defined programmatically as well.